### PR TITLE
LPS-129616 Update getUserAccountEntriesCount

### PR DESCRIPTION
**Pull request by alee8888**
@drewbrokke

### CODE REVIEW

**Comment by alee8888** [comment]
ci:test:sf

File: modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/service/test/AccountEntryLocalServiceTest.java [hunk 1]
```diff
@@ -1039,6 +1039,12 @@ private void _testGetUserAccountEntries(
 			userAccountEntries.toString(), expectedAccountEntries.size(),
 			userAccountEntries.size());
 
+		Assert.assertEquals(
+			userAccountEntries.toString(), expectedAccountEntries.size(),
```
Comment (pei-jung) [inline modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/service/test/AccountEntryLocalServiceTest.java:1043]: do we need the message param? It seems odd to use the `userAccountEntries` as message as it's not used in the assertion.

**Comment by alee8888** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
ci:forward

**Pull request closed without merging**